### ISSUE TITLE
Angular: Add ionic live reload

### ISSUE THREAD

**Issue by KevinHetzelGFL** [nxext/nx-extensions-ionic#8]
**Is your feature request related to a problem? Please describe.**
For ionic there are this commands available to run the app with live reload on android or ios:

$ ionic capacitor run ios -l --external
$ ionic capacitor run android -l --external

I realy need it for my ionic app inside nxext Workspace. At the moment I always need to run "nx run [app-name]:sync:android" and "nx run [app-name]:open:android" after changes. It takes a lot of time

**Comment by matheo** [comment]
Trying this in Sept 2024 and I was able to run it without any modification to the `capacitor.config.ts` and just extending the `run` command, and passing the arguments to `serve` instead modifying the config:

> 2. Change your project.json at the Run Task to add -l and --external to the commands and allow remote access to the serve instance
> 
> ```json
> "run": {
>       "executor": "@nxext/capacitor:cap",
>       "options": {
>         "cmd": "run"
>       },
>       "configurations": {
>         "ios": {
>           "cmd": "run ios -l --external"
>         },
>         "android": {
>           "cmd": "run android -l --external"
>         }
>       }
>     },
> ```

and then serving the application in one terminal:

```
nx run projectName:serve --host=0.0.0.0 --port=3000
```

and running the native application in the device with:

```
nx run projectName:run
```

Thanks for addressing this here!

**Comment by andregreeff** [comment]
@matheo that's interesting, but also confusing.. because `--external` is not part of the [Capacitor CLI `run` command](https://capacitorjs.com/docs/cli/commands/run), it's specifically part of the [Ionic CLI `run` command](https://ionicframework.com/docs/cli/commands/capacitor-run) that wraps the Capacitor CLI.

unless this changed in a recent version (which I don't think it has), the `@nxext/capacitor` package doesn't use the Ionic CLI, so I don't see how that flag would have any effect here?

**Comment by matheo** [comment]
@andregreeff indeed, only the `--live-reload` was needed by the Capacitor CLI and the ` --external` was omitted without a warning message. I edited my comment accordingly. Thanks!

**Comment by andregreeff** [comment]
> @andregreeff indeed, only the `--live-reload` was needed by the Capacitor CLI and the ` --external` was omitted without a warning message. I edited my comment accordingly. Thanks!

no worries, this is still good news though. :)

just for the sake of clarity, it seems that the `--live-reload` flag was first added to the Capacitor CLI in [version `6.0.0-alpha.1`](https://github.com/ionic-team/capacitor/pull/6831), with a follow up fix to adding the cleartext attribute to the `AndroidManifest.xml` file in [version `6.1.1`](https://github.com/ionic-team/capacitor/pull/7563)..

seems like the culmination of these two changes is probably what has now resolved _this_ issue now. I guess I can ditch the custom script I created for doing this. happy days!

**Comment by matheo** [comment]
@andregreeff yeah, I'm using the latest versions
